### PR TITLE
fix DateTime parse issue

### PR DESCRIPTION
**Pull request by BethanyZhou**

### CODE REVIEW

File: src/Common/CustomAttributes/GenericBreakingChangeAttribute.cs [hunk 1]
```diff
@@ -68,8 +68,11 @@ public GenericBreakingChangeAttribute(string message, string deprecateByVersion,
             this.DeprecateByVersion = deprecateByVersion;
             this.DeprecateByVersionSet = true;
 
-            this.ChangeInEfectByDate = DateTime.Parse(changeInEfectByDate);
-            this.ChangeInEfectByDateSet = true;
+            if (DateTime.TryParse(changeInEfectByDate, out DateTime result))
```
Comment (erich-wang) [inline src/Common/CustomAttributes/GenericBreakingChangeAttribute.cs:71]:
```suggestion
            if (DateTime.TryParse(changeInEfectByDate, new CultureInfo("en-US"), DateTimeStyles.None, out DateTime result))
```

**Comment by BethanyZhou** [comment]
@erich-wang Updated according to your suggestion, thanks

File: src/Common/CustomAttributes/GenericBreakingChangeAttribute.cs [hunk 2]
```diff
@@ -68,8 +69,12 @@ public GenericBreakingChangeAttribute(string message, string deprecateByVersion,
             this.DeprecateByVersion = deprecateByVersion;
             this.DeprecateByVersionSet = true;
 
-            this.ChangeInEfectByDate = DateTime.Parse(changeInEfectByDate);
-            this.ChangeInEfectByDateSet = true;
+            string[] formats = { "MM/dd/yyyy", "MM/d/yyyy", "M/dd/yyyy", "M/d/yyyy" };
```
Comment (erich-wang) [inline src/Common/CustomAttributes/GenericBreakingChangeAttribute.cs:72]: does it work with "9/27/20"?

Comment (BethanyZhou) [reply src/Common/CustomAttributes/GenericBreakingChangeAttribute.cs:72]: No...Should we support this format?

**Review by erich-wang — APPROVED** [review]

**Pull request merged**